content/build/context/dockerignore.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Dockerignore files
+description: A .dockerignore file lets you exclude files from the context using a glob pattern
+keywords: build, context, dockerignore, exclude, glob
+---
+
```

Comment:
We could add a caption saying the logic of the ignorefile is maintained on https://github.com/moby/patternmatcher/tree/main/ignorefile if users found issues or smth like that? WDYT?

Reply:
I didn't know about this! 💡 yes will add!